src/Microsoft.Health.Fhir.Api/Modules/ControlPlaneModule.cs

```diff
@@ -0,0 +1,33 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using EnsureThat;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Health.ControlPlane.Core.Features.Rbac;
+using Microsoft.Health.Extensions.DependencyInjection;
+using Microsoft.Health.Fhir.Api.Configs;
+
+namespace Microsoft.Health.Fhir.Api.Modules
+{
+    public class ControlPlaneModule : IStartupModule
+    {
+        private FhirServerConfiguration _fhirServerConfiguration;
```

Comment:
nit: readonly

Reply:
Turns out this wasn't actually being used, removed it.